sdk/tables/azure-data-tables/README.md

```diff
@@ -7,66 +7,56 @@ The Azure Data Tables client can be used to access Azure Storage or Cosmos accou
 [Source code][source_code] | [Package (PyPI)][Tables_pypi] | [API reference documentation][Tables_ref_docs] | [Samples][Tables_samples]
 
 ## Getting started
-The Azure Data Tables can be accessed using an Azure Storage or a CosmosDB account.
+The Azure Data Tables SDK can access an Azure Storage or CosmosDB account.
 
 ### Prerequisites
 * Python 2.7, or 3.5 or later is required to use this package.
 * You must have an [Azure subscription][azure_subscription] and either
-    * an [Azure storage account][azure_storage_account] or
+    * an [Azure Storage account][azure_storage_account] or
     * an [Azure Cosmos Account][azure_cosmos_account].
 
-#### Create a storage account
+#### Create account
 If you wish to create a new storage account, you can use [Azure Portal][azure_portal_create_account],
```

Comment:
I'd put these in two bullet points, Or maybe even under H5 headers to make them pop out a bit more rather than just text.